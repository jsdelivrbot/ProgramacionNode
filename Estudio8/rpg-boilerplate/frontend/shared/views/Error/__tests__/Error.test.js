/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Error from '../Error'

describe('A </Error>', function () {
  it('renders correctly', () => {
    expect(toJson(shallow(<Error />))).toMatchSnapshot()
  })
})
