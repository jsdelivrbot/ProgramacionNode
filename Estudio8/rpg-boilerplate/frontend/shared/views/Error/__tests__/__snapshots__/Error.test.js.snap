exports[`A </Error> renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "16px",
    }
  }>
  <h1>
    Sorry!
  </h1>
  <p>
    Something went horribly wrongâ€¦
  </p>
</div>
`;
